app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -148,6 +149,11 @@
             }
         }
     }
+    &.composite-bundle{
```

Comment:
Please move styles related to bundle product to `app/design/aminhtml/Magento_Bundle/web/css/source/_module.less`. Also, please add spaces before opening brackets.